tree/tree/src/TTreeSQL.cxx

```diff
@@ -15,19 +15,16 @@
 Implement TTree for a SQL backend
 */
 
-#include <Riostream.h>
+#include <iostream>
```

Comment:
Changing from Riostream.h to iostream mean that the using namespace std; is no longer there and thus might require update the code (to either add back the using statement or fully qualified the iostream usage).  [This is fine, just work :)]

Reply:
`Riostream.h` does not have `using namespace std;` statement since a long time, 
it is just 3 standard includes. And in most cases just required for `std::endl`. 
If changes does not work - compiler should immediately complain.

Reply:
it did, didn't it :)

Reply:
~8 years ago it was removed

Reply:
Right.  I misread the errors (https://lcgapp-services.cern.ch/root-jenkins/job/root-pullrequests-build/84076/console) ...